docs/LayoutSpecs.json

```diff
@@ -375,7 +375,8 @@
               "path": "DSCore.DateTime.TimeOfDay"
```

Comment:
Will we have issues with `DSCore.DateTime.TimeOfDay` not having an `iconUrl` value? Does it show the icons now?